ccd-definition/AuthorisationCaseState/systemUpdate.json

```diff
@@ -20,13 +20,6 @@
     "UserRole": "caseworker-civil-systemupdate",
     "CRUD": "CRUD"
   },
-  {
```

Comment:
why is this removed?

Reply:
This is to allow `Solicitor responds to defence` e2e testt to pass locally while using idam stub. In idam stub we have one user with all roles, so when case moved to `STAYED` state, the case was still available to user, instead of redirecting to Case List. We could change last test assertion if the stub is used, but I couldn't find a reason why we need this permission.

Reply:
I don't think it's safe to remove, for example for my 202 story. CLAIMANT_RESPONSE event leaves case in stayed state from CCD definition, we then need to process events such as notifications etc. System update user needs to see case to trigger events

Reply:
surely there are already notification events that need to happen too?